src/main/java/step04/domain/LottoNumberParser.java

```diff
@@ -0,0 +1,26 @@
+package step04.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toList;
+
+public class LottoNumberParser {
+    private static String TOKEN = ", ";
+
+    public static Lotto execute(String lottoNumbers) {
+        return Lotto.of(
+                Arrays.stream(lottoNumbers.split(TOKEN))
+                        .map(Integer::parseInt)
+                        .collect(toList())
+        );
+    }
```

Comment:
```suggestion
    private static String TOKEN = ",";

    public static Lotto execute(String lottoNumbers) {
        return Lotto.of(
                Arrays.stream(lottoNumbers.split(TOKEN))
                        .map(String::trim)
                        .map(Integer::parseInt)
                        .collect(toList())
        );
    }
```
이렇게 하면 `1,2,3,4,5,6` `1, 2, 3, 4, 5, 6` 이렇게 두 가지 케이스 모두 파싱 가능합니다.